frontend/workflows/experimentation/src/view-experiment-run.tsx

```diff
@@ -82,14 +82,20 @@ const ViewExperimentRun: React.FC<BaseWorkflowProps> = ({ heading }) => {
       <Form>
         {experiment && (
           <>
-            {experiment.properties.items.map(property => (
-              <TextField
-                key={property.label}
-                label={property.label}
-                defaultValue={propertyToString(property)}
-                InputProps={{ readOnly: true }}
-              />
-            ))}
+            {experiment.properties.items.map(property =>
+              property.urlValue !== undefined ? (
+                <Link maxWidth="500px" href={property.urlValue}>
+                  {property.label}
+                </Link>
+              ) : (
+                <TextField
+                  key={property.label}
+                  label={property.label}
+                  defaultValue={propertyToString(property)}
```

Comment:
Should we update `propertyToString` method to take care of URL case too?

Reply:
Just looked at the function. 
Since `urlValue` will always be a string, so I am not sure if we need it because all it will do is return the value. Also, dashboard links should never be `undefined`. 
Let me know if there is anything else

Reply:
I think that we should not be assuming that for every use of `urlValue` we will always have a corresponding string values. It's true for now - as you mentioned, for dashboards links it's always the case - but nothing prevents somebody else from adding a urlValue that does NOT have a corresponding string value. That's what makes me think that we should update `propertyToString` and `compareProperties` methods.

Reply:
thanks - can we also update `compareProperties` method?

Reply:
ping

Reply:
added changes to `compareProperties` and `propertyToString`